sdks/python/apache_beam/transforms/external.py

```diff
@@ -61,6 +64,25 @@
 
 DEFAULT_EXPANSION_SERVICE = 'localhost:8097'
 
+MANAGED_SCHEMA_TRANSFORM_IDENTIFIER = "beam:transform:managed:v1"
+
+_IO_EXPANSION_SERVICE_JAR_TARGET = "sdks:java:io:expansion-service:shadowJar"
+
+_GCP_EXPANSION_SERVICE_JAR_TARGET = (
+    "sdks:java:io:google-cloud-platform:expansion-service:shadowJar")
```

Comment:
Maybe combine these in a list of "supported expansion jar targets" and replace this:

https://github.com/apache/beam/blob/74c514fc39256cc19cbea93e6257ac9f23a233ad/sdks/python/gen_managed_doc.py#L102

Reply:
Done.